src/main/java/org/fcrepo/audit/InternalAuditor.java

```diff
@@ -0,0 +1,243 @@
+        if (eventType.contains(PROPERTY_CHANGED)) {
+            if (properties != null) {
+                if (properties.equals(LAST_MODIFIED)
+                        || properties.equals(LAST_MODIFIED_BY)
+                        || properties.equals(LAST_MODIFIED + "," + LAST_MODIFIED_BY)
+                        || properties.equals(LAST_MODIFIED_BY + "," + LAST_MODIFIED)) {
+
+                /* adding/removing a file updates the lastModified property of the parent container,
+                so ignore updates when only lastModified is changed */
+                    isParentNodeLastModifiedEvent = true;
+                }
+            }
+        }
+        if (!event.getPath().startsWith(AUDIT_CONTAINER_LOCATION)
+                && !isParentNodeLastModifiedEvent) {
+            createAuditNode(event);
+        }
+    }
+
+    /**
+     * Close external connections
+     */
+    @PreDestroy
+    public void releaseConnections() {
+        LOGGER.debug("Tearing down: {}", this.getClass().getCanonicalName());
+        eventBus.unregister(this);
+    }
+
+    // namespaces and properties
+    private static final String INTERNAL_EVENT = AUDIT + "InternalEvent";
+    private static final String PREMIS_EVENT = "premis:Event";
+    private static final String PROV_EVENT = PROV + "InstantaneousEvent";
+
+    private static final String CONTENT_MOD = AUDIT + "contentModification";
+    private static final String CONTENT_REM = AUDIT + "contentRemoval";
+    private static final String METADATA_MOD = AUDIT + "metadataModification";
+
+    private static final String CONTENT_ADD = EVENT_TYPE + "ing";
+    private static final String OBJECT_ADD = EVENT_TYPE + "cre";
+    private static final String OBJECT_REM = EVENT_TYPE + "del";
+
+    private static final String PREMIS_TIME = "premis:hasEventDateTime";
+    private static final String PREMIS_OBJ = "premis:hasEventRelatedObject";
+    private static final String PREMIS_AGENT = "premis:hasEventRelatedAgent";
+    private static final String PREMIS_TYPE = "premis:hasEventType";
+
+    private static final String RDF_TYPE = "rdf:type";
+
+    private static final String HAS_CONTENT = REPOSITORY + "hasContent";
+    private static final String LAST_MODIFIED = REPOSITORY + "lastModified";
+    private static final String LAST_MODIFIED_BY = REPOSITORY + "lastModifiedBy";
+    private static final String NODE_ADDED = REPOSITORY + "NODE_ADDED";
+    private static final String NODE_REMOVED = REPOSITORY + "NODE_REMOVED";
+    private static final String PROPERTY_CHANGED = REPOSITORY + "PROPERTY_CHANGED";
+
+    /**
+     * Creates a node for the audit event under the configured container.
+     *
+     * @param event
+     * @throws RepositoryException
+     */
+    public void createAuditNode(final FedoraEvent event) throws RepositoryException {
+        final String uuid = pidMinter.mintPid();
+        final JcrTools jcrTools = new JcrTools();
+        final String identifier = event.getIdentifier();
```

Comment:
This returns the UUID.  But we should be using the full URI of the node, like: https://github.com/fcrepo4/fcrepo-camel/blob/master/src/main/java/org/fcrepo/camel/processor/ProcessorUtils.java#L54